app/models/manageiq/providers/base_manager.rb

```diff
@@ -4,6 +4,10 @@ class BaseManager < ExtManagementSystem
 
     include Inflector::Methods
 
+    has_many :child_managers,
```

Comment:
Does it matter where the asscociation is defined?
I need the `:child_managers` [here](https://github.com/ManageIQ/manageiq/pull/15590#pullrequestreview-50904965) too, but for `orchestrate_destroy`, which is implemented in `ExtManagementSystem`